packages/generator/generators/mwa-generator/templates/base-template/package.json.handlebars

```diff
@@ -19,17 +19,13 @@
     "node": ">=16.18.1"
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx,mjs,cjs}": [
-      "node --max_old_space_size=8192 ./node_modules/eslint/bin/eslint.js --fix --color --cache --quiet"
+    "*.{js,ts,cjs,mjs,d.cts,d.mts,jsx,tsx,json,jsonc}": [
```

Comment:
I am concerned about the effect of biome on the *.json.
It's not a complete replacement for prettier.